chromium_src/chrome/common/chrome_constants.cc

```diff
@@ -179,6 +177,9 @@ const base::FilePath::CharType kSupervisedUserSettingsFilename[] =
     FPL("Managed Mode Settings");
 const base::FilePath::CharType kThemePackFilename[] = FPL("Cached Theme.pak");
 const base::FilePath::CharType kWebAppDirname[] = FPL("Web Applications");
+// Only use if the ENABLE_REPORTING build flag is true
```

Comment:
We should probably be setting this buildflag to false in a separate PR

Reply:
Filed an issue here https://github.com/brave/brave-browser/issues/7095